api/migrations/20200709183253-loadMemDocs.js

```diff
@@ -27,6 +27,7 @@ const OBJ_STORE_BUCKET = process.env.OBJECT_STORE_bucket_name || 'test';
 const ep = new AWS.Endpoint(OBJ_STORE_URL);
 const s3 = new AWS.S3({
   endpoint: ep,
+  sslEnabled: false,
```

Comment:
Is this necessary?

Reply:
whoops that is only for local mino testing, should be removed